docs/advanced-customization/config/monetization.md

```diff
@@ -1,6 +1,6 @@
 ---
 description: Define various monetization variables.
-position: 0
+position: 12
```

Comment:
These positions don't seem to matter as it seems that the nav links are getting sorted alphabetically. However, I just updated them anyways. Let me know if this was the wrong thing to do?

Reply:
Absolutely fine to update these